src/test/java/org/sonarlint/intellij/ui/tree/IssueTreeModelBuilderTests.java

```diff
@@ -128,9 +129,9 @@ private static LiveIssue mockIssuePointer(String path, int startOffset, String r
     return ip;
   }
 
-  private static ClientInputFile mockFile(String path) {
+  private static ClientInputFile mockFile(String path) throws IOException {
     var file = mock(ClientInputFile.class);
-    when(file.getPath()).thenReturn(path);
+    when(file.contents()).thenReturn(path);
```

Comment:
Is this expected? You return the path as the content of the file ? Is the content important for the test anyway ?

Reply:
Removed, it was not useful